src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -1044,6 +1058,46 @@ void MinimalScene::LoadConfig(const tinyxml2::XMLElement *_pluginElem)
       renderWindow->SetCameraPose(pose);
     }
 
+    elem = _pluginElem->FirstChildElement("camera_clip");
+    if (nullptr != elem && !elem->NoChildren())
+    {
+      auto child = elem->FirstChildElement("near");
+      if (nullptr != child && nullptr != child->GetText())
+      {
+        double near;
```

Comment:
I think there are macros for `near` and `far` in Windows.h, so we should be fine if we rename these variables:

* https://stackoverflow.com/questions/118774/is-there-a-clean-way-to-prevent-windows-h-from-creating-a-near-far-macro

Reply:
:bowing_woman: Thank you! Renamed in cb964f681a7e440bf76163d893f5cbef0294522d